### ISSUE TITLE
SPTimerJobState can not be used for Farm Global Timer Jobs

### ISSUE THREAD

**Issue by ChristophHannappel** [PowerShell/SharePointDsc#776]
**Details of the scenario you try and problem that is occurring:**
Tried to disable the "App State Update" Timer Job with the SPTimerJobState ressource.

The Timerjob is bound to the Farm level and has the following properties:
Displayname: App State Update
Name: job-spapp-statequery
Typename: Microsoft.SharePoint.Administration.SPAppStateQueryJobDefinition
Webapplication: <empty>

Since the ressource expects a WebAppUrl it can not find the timerjob 

> VERBOSE: [YFG-SP16-01]: LCM:  [ End    Test     ]  [[SPTimerJobState]SP-TimerJob-AppStateUpdate]  in 0.2990 seconds.
> PowerShell DSC resource MSFT_SPTimerJobState  failed to execute Test-TargetResource functionality with error message: No timer jobs found. 
> Please check the input values 
>     + CategoryInfo          : InvalidOperation: (:) [], CimException
>     + FullyQualifiedErrorId : ProviderOperationExecutionFailure
>     + PSComputerName        : YFG-SP16-01

**The DSC configuration that is using the resource:**
The WebAppUrl is the Central Administration URL.

```
            SPTimerJobState SP-TimerJob-AppStateUpdate
            {
                PsDscRunAsCredential = $SPSetupAccount
                TypeName = 'Microsoft.SharePoint.Administration.SPAppStateQueryJobDefinition'
                WebAppUrl = 'https://yfg-sp16-01:55000'
                Enabled = $false
            }
```

or with an Empty WebAppUrl Parameter
```
            SPTimerJobState SP-TimerJob-AppStateUpdate
            {
                PsDscRunAsCredential = $SPSetupAccount
                TypeName = 'Microsoft.SharePoint.Administration.SPAppStateQueryJobDefinition'
                WebAppUrl = '
                Enabled = $false
            }
```
**Version of the Operating System and PowerShell the DSC Target Node is running:**
Windows Server 2016

> Name                           Value                                                                                                                                                                                                                                                                                                                     
> PSVersion                      5.1.14393.1944                                                                                                                                                       
> PSEdition                      Desktop                                                                                                                                                              
> PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                              
> BuildVersion                   10.0.14393.1944                                                                                                                                                      
> CLRVersion                     4.0.30319.42000                                                                                                                                                      
> WSManStackVersion              3.0                                                                                                                                                                  
> PSRemotingProtocolVersion      2.3                                                                                                                                                                  
> SerializationVersion           1.1.0.1       

               

**Version of SharePoint that is used (e.g. SharePoint 2016):**
SharePoint 2016
**Version of the DSC module you're using:**
2.1.0.0

**Comment by ykuijs** [comment]
You have to specify "N/A" if it is a global job. This is also specified in the [Wiki page](https://github.com/PowerShell/SharePointDsc/wiki/SPTimerJobState) of the resource, but I see how this can be confusing. Will update the description to make it more clear.

**Comment by ChristophHannappel** [comment]
Hey, thank you for the explanation. Now it works as expected.

**Comment by ykuijs** [comment]
Included in PR #775
